features/smoke_tests/sessions.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Session functionality smoke tests
+
+Scenario: Automated sessions send
+    When I run "AutoSessionSmokeScenario"
+    And I wait to receive 2 requests
+
+    # Session payload
+    Then the request is valid for the session reporting API version "1.0" for the "Android Bugsnag Notifier" notifier
+    And the payload field "notifier.name" equals "Android Bugsnag Notifier"
+    And the payload field "sessions" is an array with 1 elements
+    And the session "user.id" is not null
+    And the payload field "sessions.0.user.id" is stored as the value "automated_user_id"
+    And the session "id" is not null
+    And the payload field "sessions.0.id" is stored as the value "automated_session_id"
+    And the session "startedAt" is not null
+
+    # App data
```

Comment:
`binaryArch` is also serialized for sessions

Reply:
I'm not seeing that present on the session payloads currently.

Reply:
Actually - I think this is only set for NDK errors IIRC.